### ISSUE TITLE
Mark Build Plates as Printed

### ISSUE THREAD

**Issue by rromo12** [bambulab/BambuStudio#3197]
When working on a project with many build plates, there's not any good way to keep track and quickly  see which build plates have already been printed

**Describe the solution you'd like**
An  Option to Right Click Build Plates in Preview/Prepare windows and "Mark As Printed"  after which selected build plate is shaded green to identify it's been printed and/or automatically mark green when sent to printer
![image](https://github.com/bambulab/BambuStudio/assets/8053975/221f6611-f237-4a22-97a3-ccec88c5c8fa)

**Describe alternatives you've considered**
Alternative: adding a simple Notes tab  with a text field that's saved per project which would be more flexible allowing other notes to be made per project file

![image](https://github.com/bambulab/BambuStudio/assets/8053975/cc20ca77-3abb-4dfa-abed-bed52498cda2)

**Comment by Eldenroot** [comment]
+1

**Comment by snassek** [comment]
I would agree with this feature request.  I have printed several projects with many plates and there isn't an easy way to keep track of what has been printing/printed.  Something as simple as an icon next to the build plate that could just click through statuses, but it could be as detailed where it would do it automatically.  

An option to have it automatically save the project to keep track would be nice too.
